run.py

```diff
@@ -1,25 +1,29 @@
 from flask_cors import CORS
 from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
 from application.utils.logging import configure_structlogger
+from application.models.models import db
```

Comment:
👍 This pattern is much more like idiomatic Flask http://flask.pocoo.org/docs/0.12/patterns/appfactories/#factories-extensions